src/Tribe/Attendees.php

```diff
@@ -752,72 +754,97 @@ public function sanitize_csv_value( $value ) {
 	 * @since 4.6.2
 	 *
 	 */
-	public function maybe_generate_csv() {
-		if ( empty( $_GET['attendees_csv'] ) || empty( $_GET['attendees_csv_nonce'] ) || empty( $_GET['event_id'] ) ) {
+	public function maybe_generate_csv( $all = false ) {
```

Comment:
Two things:

1. We'll need to update the docblock to reflect the new parameter.
2. I'm not so sure that `$all` as a boolean value is the best option here. Maybe something like `$event_id` with possible values as a string (for `'all'`) or an integer (for the event ID). Bonus would be using `is_numeric()` to ensure a string version of an ID also works in that case.